### PR TITLE
Clarify text about actions defined for DID Registry. Fix #43.

### LINKED ISSUE #43: [Introduction] Doesn't read well...
@mwherman2000 moved from CCG (https://github.com/w3c-ccg/did-spec/issues/220)

### PR DESCRIPTION
**Pull request by peacekeeper**
Fixes #43.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-spec/pull/71.html" title="Last updated on Oct 16, 2019, 9:42 AM UTC (5db853a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-spec/71/2ae47b7...5db853a.html" title="Last updated on Oct 16, 2019, 9:42 AM UTC (5db853a)">Diff</a>

### CODE REVIEW

**Comment by msporny** [comment]
> Editorial. Fixes #43.

I'm going to get super pedantic here :) -- I can't tell what you changed in this commit by the commit text, and it requires me to go and look at the issue to figure it out. Suggested commit text:

> Clarify text about actions defined for DID Registry. Fixes #43.

File: index.html [hunk 1]
```diff
@@ -246,8 +246,8 @@ <h1>
 namespace definition (such as the UUID URN namespace defined in
 [[RFC4122]]). The difference is that a DID Method specification, in
 addition to defining a specific DID scheme, also specifies the
-methods for resolving and deactivating DIDs and writing DID Documents on the
-appropriate <a>DID Registry</a>.
+methods for resolving and deactivating DIDs on and writing DID Documents to
```
Comment (TallTed) [inline index.html]:
```suggestion
methods for resolving and deactivating DIDs on, and writing DID Documents to,
```
Commas help comprehension.

A full rewrite of this sentence may be better... perhaps --
```
In addition to defining a specific DID scheme, a DID Method specification 
specifies the methods for resolving and deactivating DIDs on, and writing 
DID Documents to, the appropriate <a>DID Registry</a>.
```

**Comment by peacekeeper** [comment]
> > Editorial. Fixes #43.
> 
> I'm going to get super pedantic here :)

+1 to being pedantic in the WG. What would you recommend here?
1. Change the commit text and force-push.
2. Add another commit on top of this branch with the suggested commit text (and then squash+merge).
3. Create a new PR to replace this one, with the suggested commit text.
4. Do nothing in this PR but use more precise commit texts in the future.

I'm fine with any of the above, just let me know.

**Comment by msporny** [comment]
> +1 to being pedantic in the WG. What would you recommend here?

:)

> Change the commit text and force-push.

This is what I'd do.

> Add another commit on top of this branch with the suggested commit text (and then squash+merge).

This would also be fine.

> Create a new PR to replace this one, with the suggested commit text.

Don't do this, too much trouble.

> Do nothing in this PR but use more precise commit texts in the future.

At the very least, you or I can squash-merge and reword the commit. The cleanest thing is you fix up your local branch and force push to this PR.

**Pull request merged**